index.html

```diff
@@ -60,7 +61,7 @@ <h4>Pro</h4>
             <a href="#"><button>Get Started</button></a>
           </div>
           <div class="pricing__card">
-            <img src="assets/img/pro.svg" alt="enterprise" />
+            <img src="assets/img/enterprise.svg" alt="enterprise" />
```

Comment:
this was already fixed in #30
Did you commit directly to main then?

Reply:
its weird because in my local file there's the right name, let's see at the next push